app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/service_building/AddServiceBuildingType.kt

```diff
@@ -28,6 +31,9 @@ class AddServiceBuildingType : OsmFilterQuestType<ServiceBuildingType>() {
 
     override fun createForm() = AddServiceBuildingTypeForm()
 
+    override fun getHighlightedElements(element: Element, getMapData: () -> MapDataWithGeometry) =
+        getMapData().filter("ways, relations with building ~ service|transformer_tower")
```

Comment:
Good idea! Although I would extend it, as bigger power substations are not in `building` (but outside in `landuse=industrial` - although that [specific location](https://www.openstreetmap.org/way/257031979) doesn't have that tag). So maybe something more generic like:

```suggestion
        getMapData().filter("ways, relations with building ~ service|transformer_tower or power")
```

of course there are possibly other service building inside "service" areas e.g. (waste)water processing? 
We could add them too (needs looking up what types can be in `building=service` and thus likely around it too)